aprslib/parsing/__init__.py

```diff
@@ -116,6 +116,7 @@ def parse(packet):
     # capture ParseErrors and attach the packet
     except (UnknownFormat, ParseError) as exp:
         exp.packet = packet
+        exp.parsed = parsed
```

Comment:
Whats the purpose of this?

Reply:
This got beyond the scope of the original pull request, but being very new to multi-user git, this ended up here.

When we have a packet with an unknown or unsupported body type, this allows us to return the valid parsed header data, rather than having an all-or-nothing parsing.  This is particularly important for third-party packets, where the third-party packet is well-formed but the original one is either malformed or unsupported.

For a project I'm working on, I'm interested as much in the headers as in the data, so this allows me to use packets where the path is well-known but the message is malformed or unsupported.  It seems more efficient to return the partial decode than to return an error and have to then catch it and parse just the header again.